src/databricks/labs/ucx/install.py

```diff
@@ -368,6 +368,7 @@ def folder(self):
 
     def run(self):
         logger.info(f"Installing UCX v{PRODUCT_INFO.version()}")
+        self._enable_files_in_repos()
```

Comment:
Can you move this to WorkspaceInstaller, before you read the current configuration file